pallets/subtensor/src/lib.rs

```diff
@@ -1203,14 +1203,21 @@ pub mod pallet {
                 "feabaafee293d3b76dae304e2f9d885f77d2b17adab9e17e921b321eccd61c77"
             ];
             weight = weight
+                // Initializes storage version (to 1)
                 .saturating_add(migration::migrate_to_v1_separate_emission::<T>())
+                // Storage version v1 -> v2
                 .saturating_add(migration::migrate_to_v2_fixed_total_stake::<T>())
+                // Doesn't check storage version. TODO: Remove after upgrade
```

Comment:
Why do we have to remove this after the upgrade ? this was added ages ago and wont run anymore since the storage version would be higher

Reply:
This migration doesn't check or update the version. Instead, it checks if root network has already been created and exits if it has. So, there's no harm in not removing it, except added weight in `on_runtime_upgrade`.